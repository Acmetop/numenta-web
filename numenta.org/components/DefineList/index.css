/**
 * Numenta.org HTM Community website source code
 * MIT License (see LICENSE.txt)
 * Copyright © 2005—2016 Numenta <http://numenta.com>
 */

.horizontal {
  composes: cf from 'tachyons';
  composes: lh-title from 'tachyons';
}

.vertical {
  composes: lh-copy from 'tachyons';
  composes: mb3 from 'tachyons';
  composes: measure from 'tachyons';
  composes: measure-wide-l from 'tachyons';
}


.defineList {
  composes: ma0 from 'tachyons';
  composes: pa0 from 'tachyons';
}
