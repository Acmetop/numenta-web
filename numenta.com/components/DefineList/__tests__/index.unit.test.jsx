import React from 'react'
import renderer from 'react-test-renderer'

import DefineList from '../../DefineList'


describe('<DefineList /> React component unit test suite', () => {

  it('Renders correctly', () => {
    const component = renderer.create(
      <DefineList />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

})
