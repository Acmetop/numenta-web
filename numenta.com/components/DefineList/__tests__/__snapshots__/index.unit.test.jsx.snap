exports[`<DefineList /> React component unit test suite Renders correctly 1`] = `
<dl
  className="defineList vertical" />
`;
